docs/en/helm/chart-as-capability.md

```diff
@@ -0,0 +1,146 @@
+# Use Helm chart as capability module
+
+Here is an example of how to use Helm chart as workload capability module.
+
+## Install fluxcd/flux2 as dependencies
+
+This feature depends on several CRDs and controllers from [fluxcd/flux2](https://github.com/fluxcd/flux2), so we prepared a simplified Helm chart to install dependencies.
+
+It's worth noting that, flux2 doesn't offer an official Helm chart to install.
+And this chart only includes minimum dependencies this feature relies on, not all of flux2.
+
+```shell
+	helm install --create-namespace -n flux-system helm-flux http://oam.dev/catalog/helm-flux2-0.1.0.tgz
+```
+
+## Write WorkloadDefinition 
+Here is an example `WorkloadDefinition` with only required data of a Helm module.
+
+Comparing to existing workload definition based on CUE template, several points worth attention in Helm module.
+
+- `.spec.definitionRef` is required to indicate the workload GVK in your Helm chart. For example, in our sample chart, the core workload is `deployments.apps/v1`.
+- `.spec.schematic.helm` contains information of Helm release & repository.
+
+```yaml
+apiVersion: core.oam.dev/v1alpha2
+kind: WorkloadDefinition
+metadata:
+  name: webapp-chart
+  annotations:
+    definition.oam.dev/description: helm chart for webapp
+spec:
+  definitionRef:
+    name: deployments.apps
+    version: v1
+  schematic:
+    helm:
+      release:
+        chart:
+          spec:
+            chart: "podinfo"
+            version: "5.1.4"
+      repository:
+        url: "http://oam.dev/catalog/"
+```
```

Comment:
Put the example first before explaining `.spec.definitionRef` and `.spec.definitionRef`.